benchexec/tablegenerator/react-table/src/components/SelectColumn.js

```diff
@@ -219,6 +253,7 @@ export default class SelectColumn extends React.Component {
             Apply and close
           </button>
           <input />
+          {this.backButton()}
```

Comment:
Is the `render` method really the correct time in the life cycle of react components to setup these handlers?